benchmarks/superphot_plus/test_fit.py

```diff
@@ -5,39 +5,33 @@
 
 from superphot_plus.ztf_transient_fit import dynesty_single_file
 from superphot_plus.fit_numpyro import numpyro_single_file
-from superphot_plus.import_ztf_from_alerce import (
-    generate_single_flux_file,
-    save_datafile,
-    import_lc,
-)
+from superphot_plus.import_ztf_from_alerce import save_datafile, import_lc
+
+test_sn = "ZTF22abvdwik"  # can change to any ZTF supernova
 
 OUTPUT_DIR = "benchmarks/data/"
 
-test_sn = "ZTF22abvdwik"  # can change to any ZTF supernova
 lc_fn = os.path.join(OUTPUT_DIR, test_sn + ".csv")
 fn_to_fit = os.path.join(OUTPUT_DIR, test_sn + ".npz")
 
 
 @pytest.fixture(scope="session", autouse=True)
 def setup():
-    """Generates sample data file for benchmarking purposes."""
-    os.makedirs(OUTPUT_DIR, exist_ok=True)
-    generate_single_flux_file(test_sn, OUTPUT_DIR)
-    # Save the preprocessed lightcurves
+    """Generates the compressed sample file for benchmarking purposes."""
     t, f, ferr, b, _, _ = import_lc(lc_fn)
     save_datafile(test_sn, t, f, ferr, b, OUTPUT_DIR)
```

Comment:
I'm not sure this step is needed. Can't you just pass `test_sn` directly into the functions below?